docs/releases.md

```diff
@@ -4,23 +4,71 @@
 
 ### Overview
 
-Welcome to **April 2021 release** :octicons-heart-16: of **DietPi**. This release is focused to fine tune existent DietPi tools, making them easier to use, with more options.  
+Welcome to **April 2021 release** :octicons-heart-16: of **DietPi**. We have a number of updates in this version that we hope you will like. Some of the key highlights include:
 
-![DietPi version 7](assets/images/dietpi-version7.1.jpg){: width="300" }
+!!! hint "Highlights"
+    
+    - [Merged software list](#select-software-from-a-merged-list) - we simplified the software selection, using a single list ! You have now more options to choose from ! The list contains over 190 software options !
+    - [New software](#new-optimised-software-in-71) - IPFS Nodes, Common UNIX Printing System (CUPS), and more in programming (Go language, and an excellent IDE: VSCodium)
+    - [DietPi tools: New additions & notable updates](#dietpi-tools-in-71-new-notable-updates) - DietPi supports more VPN clients and it has more Dynamic DNS options. Enabling SSL/TLS certificates is easier now using the updates brought to Let's Encrypt tool.
+    - [Improvements](#changes-improvements-optimisations-in-71) - More improvments in DietPi tools (to name just a few: DietPi-LetsEncrypt, DietPi-Drive_Manager, DietPi-Config etc.) have been updated, along with more changes to more software packages.
+    - [Bug fixes](#bug-fixes) & updates to [supported SBC](#supported-sbc-updates-in-71) list
 
-[_Source: Pexels.com_](https://www.pexels.com/photo/abstract-aluminum-architectural-architecture-210158)
+Many thanks to all contributors and beta testers :octicons-heart-16: :octicons-heart-16: !! With your help we could achieve all of these updates ! Install or upgrade to this latest update - it has more things we hope you would enjoy ! Share your feedback on [Twitter](https://twitter.com/DietPi_), [DietPi Forum](https://dietpi.com/phpbb/viewforum.php?f=9&sid=e6fcb5de3776c2d8eb1b22cc1fb24ca5), and report any issue to [GitHub](https://github.com/MichaIng/DietPi/issues) !
 
-### Merged software list
+As a new thing, we already scheduled the next update. DietPi 7.2 will be launched on Friday, 27 May !
```

Comment:
It will be Saturday 29 May 😃. Saturday is a good day for me to do last reviews and tests.

Reply:
:) very sorry, I'm changing it now !!